localization/localization_error_monitor/launch/localization_error_monitor.launch.xml

```diff
@@ -2,7 +2,7 @@
   <arg name="input/odom" default="/localization/kinematic_state"/>
   <arg name="param_file" default="$(find-pkg-share localization_error_monitor)/config/localization_error_monitor.param.yaml"/>
 
-  <node name="localization_error_monitor" exec="localization_error_monitor" pkg="localization_error_monitor" output="screen">
+  <node pkg="localization_error_monitor" exec="localization_error_monitor_node" output="screen">
```

Comment:
Similarly, please change this from `screen` to `both`.

Reply:
I changed the log output from "screen" to "both" in https://github.com/autowarefoundation/autoware.universe/pull/7102/commits/258a593ce0ffd405226aa8663174b79c0c2f854b